llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -4213,3 +4213,10 @@ define void @foo() {
   EXPECT_EQ(NewUIEnd->getParent(), BB);
   EXPECT_EQ(NewUIEnd->getNextNode(), nullptr);
 }
+
+/// Makes sure that all Instruction sub-classes have a classof().
+TEST_F(SandboxIRTest, CheckClassof) {
+#define DEF_INSTR(ID, OPC, CLASS)                                              \
+  EXPECT_NE(&sandboxir::CLASS::classof, sandboxir::Instruction::classof);
```

Comment:
nit: use `&` on `sandboxir::Instruction::classof` too, to make it consistent with the other side of the comparison?